### ISSUE TITLE
one-line install CMD can fail if %path% contains ( )

### ISSUE THREAD

**Issue by teknowledgist** [chocolatey/chocolatey.org#412]
### What You Are Seeing?
Odd error(s) ("intel\iCLS unexpected at this time") and Chocolatey itself doesn't install when running the one-line CMD.

### What is Expected?
Chocolatey installs.

### How Did You Get This To Happen? (Steps to Reproduce)
After a hundred or so successful Chocolatey installs, I encountered a computer with a path variable containing parenthesis -- specifically it included "Program Files (x86)".  

### How to fix the problem
Change the one-line CMD install instruction to include quotes around setting the path:

@powershell -NoProfile -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"

**Comment by ferventcoder** [comment]
Interesting. I've never run into this issue. I wonder if it is malformed parentheses (meaning only "`(`" or "`)`".

**Comment by teknowledgist** [comment]
I found it interesting too.  :-)

I didn't notice that %path% was malformed on the machine (and I don't have access any longer), but enclosing the set command subject in quotes did fix the issue.  I found the solution suggestion somewhere on stackoverflow referenced as "extended syntax of SET", but I haven't had much luck finding an official description.

I should add that I was using the "one-line" install inside of a larger script, but it is the same script I've used on many other computers.  I don't see why it would matter functionally, but I was not just pasting the line into a cmd window.
